README.md

```diff
@@ -1,4 +1,4 @@
-<h3 align="center">😻 In-App Subscriptions Made Easy 😻</h1>
+<h3 align="center">😻 In-App Subscriptions Made Easy 😻</h3>
```

Comment:
Oops 😳

Reply:
Still worked...

Reply:
I wonder if we should use `###` instead?

Reply:
I don't think (?) you can center without html on github

Reply:
Oooh I didn't realize this is centered too, never mind.